### PR TITLE
Django links

### PR DESCRIPTION
**Pull request by will-moore**
Small doc fixes I noticed:

 - Update Django doc links to 1.11 (teach some links to use the : djangodoc: helper)
 - Remove Unicode section from BlitzGateway page - no longer relevant.

### CODE REVIEW

**Comment by will-moore** [comment]
https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/116/console is failing with:

```
00:57:43      [exec] (line 2213) broken    https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#django.core.urlresolvers.reverse - Anchor 'django.core.urlresolvers.reverse' not found 
```

**Comment by jburel** [comment]
Failed travis with 
```
     [exec] Warning, treated as error:
     [exec] /home/travis/build/ome/ome-documentation/omero/developers/Web/CreateApp.rst:2:Duplicate explicit target name: "omero-web-apps-examples".
```

File: omero/developers/Web/CreateApp.rst [hunk 1]
```diff
@@ -113,8 +114,8 @@ page which will display your Name and list your Projects.
 Exploring the app
 -----------------
 
-The ``minimal_webapp`` code is well documented to explain
-how the app is working.
+The `minimal_webapp code <https://github.com/ome/omero-web-apps-examples/tree/master/minimal-webapp>`_
+is well documented to explain how the app is working.
```
Comment (mtbc) [inline omero/developers/Web/CreateApp.rst]:
```suggestion
is well-documented to explain how the app is working.
```

File: omero/developers/Web/ReleaseApp.rst [hunk 2]
```diff
@@ -24,7 +24,8 @@ Configuring your app name and label
 -----------------------------------
 
 We support the option of configuring your OMERO.web app with a name and label.
-See Django `Configuring Applications <https://docs.djangoproject.com/en/1.8/ref/applications/#configuring-applications>`_.
+:djangodoc:`Configuring Applications <ref/applications/#configuring-applications>`.
```
Comment (mtbc) [inline omero/developers/Web/ReleaseApp.rst]:
```suggestion
See :djangodoc:`Configuring Applications <ref/applications/#configuring-applications>`.
```

**Comment by mtbc** [comment]
`config.rst` is also fixed in https://github.com/ome/omero-web?

**Comment by mtbc** [comment]
Staging includes,
- https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/developers/Web/CreateApp.html
- https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/developers/Web/ReleaseApp.html
- https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-docs/lastSuccessfulBuild/artifact/omero/_build/html/sysadmins/config.html#web

**Comment by will-moore** [comment]
Links fixed in omero-web at https://github.com/ome/omero-web/pull/103/files#diff-aa8789d92bc0441643a6535447739dbd

**Review by mtbc — APPROVED** [review]

File: omero/sysadmins/config.rst [hunk 3]
```diff
@@ -2210,7 +2210,7 @@ Default: `None`
 
 omero.web.login_redirect
 ^^^^^^^^^^^^^^^^^^^^^^^^
-Redirect to the given location after logging in. It only supports arguments for :djangodoc:`Django reverse function <ref/urlresolvers/#django.core.urlresolvers.reverse>`. For example: ``'{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": {"experimenter": -1}}'``
+Redirect to the given location after logging in. It only supports arguments for :djangodoc:`Django reverse function <ref/urlresolvers/#reverse>`. For example: ``'{"redirect": ["webindex"], "viewname": "load_template", "args":["userdata"], "query_string": {"experimenter": -1}}'``
```
Comment (joshmoore) [inline omero/sysadmins/config.rst:2213]: @will-moore : these will need back porting to omero-web.

Comment (will-moore) [reply omero/sysadmins/config.rst:2213]: Done - see https://github.com/ome/omero-web/pull/103/files#diff-aa8789d92bc0441643a6535447739dbd

Comment (joshmoore) [reply omero/sysadmins/config.rst:2213]: Ah, ta. Then I should have done an OMERO.web release for the 5.6.0-rc1 docs. Sorry about that.

**Pull request merged**